### DISCUSSION TITLE
enhancement request

### DISCUSSION THREAD

**Discussion by silverspr** [DirectoryLister/DirectoryLister#1087] [Ideas & Features]
Suggest Pagination on folders/files and search findings would be a nice feature.  Perphaps limit the number of items to display per page like 'datatables'

**Comment by PHLAK** [comment]
Thank you for the suggestion. We do not plan on adding pagination at this time. Directory Lister is intended to mimic a listing of a directory, showing all files without hiding any behind a paginated listing.
